src/styles/main.scss

```diff
@@ -21,6 +21,7 @@ body {
   display: flex;
   align-items: center;
   justify-content: center;
+  cursor: pointer;
```

Comment:
The cursor property is set to pointer, which is appropriate for clickable elements. Ensure that the JavaScript logic correctly handles clicks on elements with this class.